src/ExadelMentorship.WebApi/Controllers/MessageController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using ExadelMentorship.WebApi.RabbitMQ;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ExadelMentorship.WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MessageController : ControllerBase
+    {
+        private readonly IMessageProducer _messagePublisher;
+
+        public MessageController(IMessageProducer messagePublisher)
+        {
+            _messagePublisher = messagePublisher;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> CreateOrder(string message)
+        {
+            _messagePublisher.SendMessage(message);
```

Comment:
Is there an async version of message sending, because it is IO bound operation and we want this action not to be blocking.

Reply:
.SendMessage is my created method, which makes publishing to rabbitMQ, as checked 'channel.BasicPublish' does not have async version